### ISSUE TITLE
Reindex All Job running out of Time

### ISSUE THREAD

**Issue by martinlovell** [yalelibrary/YUL-DC#1132]
On staging, the reindex all job is running out of time to complete:

```
execution expired (Delayed::Worker.max_run_time is only 3600 seconds)
```

Acceptance:
- [x] Plan this to batch jobs into about 5000 each and then have it automatically kick off the next one

Notes:
- Could find a way to speed up the process
- Eventually the jobs will be to large you won't want to index everything at once; batch into groups of a couple thousand?
     - Each batch triggers the next till it runs out

**Comment by martinlovell** [comment]
https://github.com/yalelibrary/yul-dc-management/pull/564

**Comment by martinlovell** [comment]
This has been tested on collections-test by running reindex and spot checking that all solr records had the timestamp updated.

Testing procedure:
After running reindex, keep an eye on the working delayed job and could see the Reindex job in the queue.  After a few minutes, you will see a similar Reindex job in the queue but with a different job ID and a parameter of 5000 (or some multiple of 5000 if you miss the second one.)  These reindex jobs will run in batches of 5000 until all the oids are reindexed.
Afterwards, check some oids at the beginning and end or the parent objects table to make sure the solr records' timestamps have been updated.  (View the solr record from Solr by clicking the Solr Record link at the bottom of the parent object show page.)

**Comment by FCRodriguez7** [comment]
Released management v2.35.0
